cmake/external/openvino.cmake

```diff
@@ -0,0 +1,26 @@
+include (ExternalProject)
+
+set(OPENVINO_URL https://github.com/opencv/dldt.git)
+set(OPENVINO_TAG 2018)
```

Comment:
2018 is a branch right? is there a tag we can fix to instead?
(noticed there are 2019 branch and tags, would we want to take those later versions instead?)

Reply:
Yes we can fix to the tag 2018_R5. I will make this change. We will be supporting this version initially. We will move to 2019_R1 a little later.

Reply:
i think we're not supporting building openvino from source for now, right? if so, please add some comments indicating that.

Reply:
Will do

Reply:
Added suggested changes